templates/zerver/help/getting-started-with-zulip.md

```diff
@@ -0,0 +1,58 @@
+# Getting started with Zulip
+
+Welcome to Zulip!  Like any new tool, mastering Zulip will take some time,
+but once you get the hang of it, you'll never want to use a chatroom tool
```

Comment:
I think the idea is that we're distinguishing Zulip (a topic-based tool) from things like Slack and IRC where the only granularity is the individual channel (i.e. a chatroom where all the messages get dumped in a confusing way), and using "chatroom apps" as kind of a catch-all for Slack, IRC, Hipchat, and all the other similar competitors.